packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -174,8 +174,8 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
         sourceFile: sourceDataResourceFile,
         projectFile: dataResourceFile,
         deployThresholdSec: {
-          onWindows: 40,
-          onOther: 30,
+          onWindows: 130,
+          onOther: 120,
```

Comment:
deploy time increased after adding custom operations with JS resolver handlers

Reply:
talked to Ivan. Looks like we're constrained by CFN here. The base case (without custom JS resolvers) seem to be still meeting the benchmark. If the base case increases, this overall number will also be bumped. So we'll detect an issue if the deployment takes longer but it'll be harder for us to debug where the issue is. In the API deployment or the resolver.

Reply:
@iartemiev I think we should just create new test project that has only data. So that we can guard simple case with previous thresholds. And have a project where data features keeps added.

Compounding multiple features into single test project will make us blind. I.e. we won't be able to detect regression of simpler setups.